### ISSUE TITLE
Cannot upload Taxi-v2 results

### ISSUE THREAD

**Issue by bestxolodec** [openai/gym#507]
## Problem

In last version of `gym==0.7.3` there is no more `Taxi-v1` env because of #345. 
But it is either not possible to upload results for `Taxi-v2` , i.e. `gym.upload(...)` raises
```
InvalidRequestError: Request req_RmFUYGHlT2qqNm6KZEaGww: This version of the environment is not registered on the server, but other versions exist. You may need to upgrade your version of gym. Env Taxi-v2 not found (valid versions include ['Taxi-v1']) 
```
I guess something need to be updated on the server. 

## Steps to reproduce 

```
import gym
env = gym.make("Taxi-v2")
env = gym.wrappers.Monitor(env, directory="videos", force=True)
o = env.reset()
for t in range(100):
    o, r, done, _ = env.step(env.action_space.sample())
    if done: 
        break 
env.close()
gym.upload("videos",api_key=TOKEN)
```

## PS 
There is possibly also a problem with video recording for new environement Taxi-v2. 
Currently i get no mp4 file in a directory i am writing to (only `json` ones). But I guess I could be missing something.

**Comment by tlbtlbtlb** [comment]
@jietang Is the deployed scoreboard lagging the gym release here?

**Comment by LecJackS** [comment]
About the PS up there, is that a real issue? or a misconfiguration?

I recorded a couple of environments from a jupyter notebook and all of them only saved .json files, not mp4.

**Comment by tlbtlbtlb** [comment]
While graphical environments produce an `mp4`, text environments like Taxi produce an [ASCIIcast file](https://github.com/asciinema/asciinema/blob/master/doc/asciicast-v1.md) which are a type of JSON file.

**Comment by LecJackS** [comment]
Ok, I feel really ashamed right now. Thank you very much for you response.

**Comment by Taygrim** [comment]
Why is this closed? I too cannot upload taxi-v2 solution. I get the same error:
This version of the environment is not registered on the server, but other versions exist. You may need to upgrade your version of gym. Env Taxi-v2 not found (valid versions include ['Taxi-v1'])
And the only version of taxi is https://gym.openai.com/envs/Taxi-v1

**Comment by jenleong** [comment]
Still getting same error as Taygrim when I try to upload taxi-v2 solution. Please fix, as Taxi is unusable right now.

TL;DR:   
Taxi-v1 is deprecated, but Taxi-v2 is not supported for upload.
